tilauskasittely/tilaus-valmis.inc

```diff
@@ -1641,7 +1656,8 @@ else {
               ) and ($maehrowx["jv"] == '' or $totarowx["tulostustapa"] != 'H') and $laskurow["splittauskielto"] != "E"
             ) or
             ($row["kerayspvm"] != $edkerayspvm and $edkerayspvm != 'x' and $edomalle_tilaukselle == 0 and $yhtiorow["splittauskielto"] == "K") or
-            ($omalle_tilaukselle != $edomalle_tilaukselle and $edomalle_tilaukselle != 'x')
+            ($omalle_tilaukselle != $edomalle_tilaukselle and $edomalle_tilaukselle != 'x') or
+            ($vakit_eri_tilaukselle and $row["vakkoodi"] != "" and !$tilaus_splitattu)
```

Comment:
@tiere Miksi tässä on "and !$tilaus_splitattu"-ehto? Meneekö nyt homma oikein jos tilaan tavaraa esim. kolmesta eri varastosta, sillä tavalla, että tilaan jokaisesta varastosta yhden normaalin sekä yhden vaarallisen tuotteen? Pitäisi siis syntyä kuusi tilausta. Syntyykö?